### PR TITLE
manifests/9.2: Use RHEL 9.2 repos & cleanups

### PR DESCRIPTION
**Pull request by travier**
Revert "manifest/9.2: Ensure package come from RHEL by default"

We already have to use coreos-installer from the RHAOS repo and it's
likely that we will have to do it again in the future for 4.14.

Remove this to avoid having to make a PR for each package.

This reverts commit 49f9824db2b6f929567d2d41cea6c97df766cf37.

---

manifest/9.2: Keep legacy coreos/toolbox

Unfortuantely, we're not yet ready to move to the new containers/toolbox
workflow. We'll attempt that for a future release.

See: https://issues.redhat.com/browse/OCPBUGS-8009
Partially reverts: https://github.com/openshift/os/commit/65776fa6e94fc25d73fd32a26f38547d55f61d44

---

manifests: Merge common-el9.yaml into common.yaml

---

manifests/9.2: Use RHEL 9.2 repos

---

manifest/c9s: Update for 4.14

---

manifest/c9s: Disable all repos by default

---

manifests: Remove RHEL 8 manifests

---

manifests: Remove RHEL 9.0 manifests

---

manifest: Back to rhel toolbox

Since we don't have a legacy toolbox build in OCP channels yet.

---

rhel-9.2: Fix two more references to 9.0

With this things build for me.

---

### CODE REVIEW

**Comment by cgwalters** [comment]
OK I pushed two additional commits here.

File: common.yaml [hunk 1]
```diff
@@ -165,6 +167,43 @@ postprocess:
...
+     # generating it.
+     ln -sr /usr/share/zoneinfo/UTC /etc/localtime
+  - |
+    #!/usr/bin/env bash
+    set -xeuo pipefail
+    # Backport https://github.com/dracutdevs/dracut/commit/25a92885a9519701cc480298c2b082e2e2bf5ebe
+    s=/usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
+    if test -f "$s"; then
+      chmod a+x "$s"
+    fi
```
Comment (HuijingHei) [inline common.yaml:206]: We can remove workaround for `nvmf-autoconnect.sh`, as the issue does not exist on 9.2
```
[cloud-user@citest-1 ~]$ rpm -q dracut
dracut-057-21.git20230214.el9.x86_64
[cloud-user@citest-1 ~]$ ll /usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
-rwxr-xr-x. 1 root root 140 Jun 19  2022 /usr/lib/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
[cloud-user@citest-1 ~]$ cat /etc/os-release 
...
REDHAT_SUPPORT_PRODUCT_VERSION="9.2 Beta"
```

Comment (cgwalters) [reply common.yaml:206]: Hmm yes...though this is in the FCOS config, so fixing that would require a cycle there...let's just keep this on our list of things to clean up later.

File: manifest-rhel-9.2.yaml [hunk 2]
```diff
@@ -20,7 +20,8 @@ repos:
   - rhel-9.2-baseos
   - rhel-9.2-appstream
   - rhel-9.2-fast-datapath
-  - rhel-9.2-server-ose-4.13
+  - rhel-9.2-server-ose-4.13  # FIXME a few things are built only in 4.13 right now, like podman
```
Comment (jlebon) [inline manifest-rhel-9.2.yaml:23]: Do we have a tracker for this we could paste here?

**Review by jlebon — APPROVED** [review]
Overall LGTM. A lot of the commits should probably be squashed to make a cleaner history, but fine as is too.

/approve
/lgtm

**Comment by cgwalters** [comment]
/override ci/prow/rhcos-92-build-test-metal 
/override ci/prow/rhcos-92-build-test-qemu
beta content isn't available yet

File: extensions-rhel-9.2.yaml [hunk 3]
```diff
@@ -53,7 +53,6 @@ extensions:
     architectures:
       - x86_64
     repos:
-      # - rhel-9.0-appstream
-      - sig-virtualization
+      - rhel-9.2-appstream
```
[the remark this answers is not in the document]
Comment (etrunko) [reply extensions-rhel-9.2.yaml:56]: Yeah, no kata-containers in rhel, only c9s.

[the remark this answers is not in the document]
Comment (cgwalters) [reply extensions-rhel-9.2.yaml:56]: C9S *is* RHEL (9).  Having a SIG doesn't mean it's "in C9S", only *associated* with it.

[the remark this answers is not in the document]
Comment (etrunko) [reply extensions-rhel-9.2.yaml:56]: Sorry, I meant C9S SIG, you are right.

[the remark this answers is not in the document]
Comment (etrunko) [reply extensions-rhel-9.2.yaml:56]: https://mirror.stream.centos.org/SIGs/9-stream/virt/x86_64/kata-containers

**Comment by cgwalters** [comment]
Argh. FTR I accidentally force-pushed over the original PR branch here.  I don't think this matters now that it's merged, but just for historical reference.

**Pull request merged**